DougieDev/whats-cookin-dougie-rachel
# Rw header click functionality

#### What changes/additions are a part of this PR?
-All of these changes relate to the DOM, adding functionality for the following: 
- When click a filter recipes category, only the recipes in that category display
- When click "My Favorite Recipes", the user's favorite recipes display
- When click "Recipes to Cook", the user's recipes to cook display
- Heart & cookbook Icon status (active v inactive) now also persists throughout navigation of the site 

#### Any background context you want to provide?
#### Where should the reviewer start?
Scripts.js has the majority of the important changes 
#### How should this be tested (test suite/manually)?
Run index.html & play around with the site, add console.log statements to check data model 
#### What are the relevant tickets?
Ticket #29 
#### Screenshots (if helpful)
#### Any questions/additional comments:
